### DISCUSSION TITLE
Can deidentified data from mimic be used to prompt large language models through external APIs?

### DISCUSSION THREAD

**Discussion by Ahmed-Medhat-Zayed** [MIT-LCP/mimic-code#1764] [MIMIC-General]
Hi

In my research, I study how can large language models be utilized to support the clinical decision-making process. Is it possible to use case notes and data from MIMIC to prompt LLMs as OpenAI through APIs (not ChatGPT)? These data shared with the model will not be used for training purposes according to the Data Processing Agreement we signed with OpenAI.

**Comment by alistairewj** [comment]
Please see the guidance here: https://physionet.org/news/post/gpt-responsible-use

> We have received inquiries regarding the use of credentialed data (MIMIC-III, MIMIC-IV, MIMIC-CXR) with online services such as GPT. The [PhysioNet Credentialed Data Use Agreement](https://physionet.org/about/licenses/physionet-credentialed-health-data-license-150/) explicitly prohibits sharing access to the data with third parties, including sending it through APIs provided by companies like OpenAI, or using it in online platforms like ChatGPT.
> 
> If you are interested in using the GPT family of models, we suggest using one of the following services:
> 
> [Azure OpenAI service](https://azure.microsoft.com/en-us/products/cognitive-services/openai-service/). You'll need to opt out of human review of the data, as (1) you are processing sensitive data where the likelihood of harmful outputs and/or misuse is low, and (2) you do not have the right to permit Microsoft to process the data for abuse detection due to the data use agreement you have signed. The form for opting out of the review process is available here: https://aka.ms/oai/additionalusecase
> [Amazon Bedrock](https://aws.amazon.com/bedrock/). Bedrock provides options for fine-tuning foundation models using private labeled data. After creating a copy of a base foundation model for exclusive use, data is not shared back to the base model for training.
> 
> If you have any questions about this policy, feel free to reach out: https://physionet.org/about/#contact_us
